src/rdb.c

```diff
@@ -3020,10 +3020,9 @@ int rdbLoadRioWithLoadingCtx(rio *rdb, int rdbflags, rdbSaveInfo *rsi, rdbLoadin
             if ((dbid = rdbLoadLen(rdb, NULL)) == RDB_LENERR) goto eoferr;
             if (dbid >= (unsigned)server.dbnum) {
                 serverLog(LL_WARNING,
-                          "FATAL: Data file was created with a Redis "
-                          "server configured to handle more than %d "
-                          "databases. Exiting\n",
-                          server.dbnum);
+                          "FATAL: Data file was created with a %s server configured to "
+                          "handle more than %d databases. Exiting\n",
+                          server.extended_redis_compat ? "Redis" : "Valkey", server.dbnum);
```

Comment:
Hi @zuiderkwast , Do you think we have to update macro instead of config here as well?

Reply:
Yes, SERVER_TITLE. It was decided in #274 that this config should not affect logging.

```suggestion
                          SERVER_TITLE, server.dbnum);
```